web/src/pages/ChatAgent/components/ExportPreviewModal.css

```
@@ -472,17 +476,28 @@
 /* react-to-print prints the hidden source div via the browser's native engine.
    These rules ensure the PDF has light-mode colors and proper typography. */
 @media print {
+  /* Must stay in step with `A4_PAGE_RULE` (lib/shellPdf.ts), which is the same
+     geometry for the routes that can inject it. This copy covers the one that
+     cannot: a plain Cmd+P of this document while the preview is up. The
+     reasoning behind `!important` is on that constant. */
   @page {
-    size: A4;
-    margin: 15mm; /* Paged.js @page{margin:0} is scoped to media=screen during print */
+    size: A4 !important;
```

Stylelint (17.14.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)